main.nf

```diff
@@ -8,18 +8,20 @@ include { humann; humann_regroup; humann_rename } from './processes/humann.nf'
 
 
 workflow {
-    
-    read_pairs_ch = Channel
-        .fromFilePairs("$params.readsdir/$params.filepattern", size: 2)
+    read_ch = Channel
+        .fromPath("$params.readsdir/$params.filepattern")
+            .map {
+                tuple(it.name.replace(".fastq.gz", ""), it)
```

Comment:
This has to change too, to match the paremeter on `XXX-params.yaml`, right?

Reply:
No, `params` comes from using the `-params-file local-params.yaml` or `-params-file aws-params.yaml`

Reply:
no, I mean the replace. If the pattern on the `-params-file` is `"*.fastq"`, then here it should be `tuple(it.name.replace(".fastq", ""), it)`, no? Anyway, minor.

Reply:
Oh I see... yes, you're right. 

For now, can probably do `it.name.replace(".fastq", "").replace(".gz", "")` to handle both....

Reply:
alternatively, `gzip *.fastq` in your input directory before running things :laughing: